src/czml3/widget.py

```diff
@@ -74,15 +98,37 @@
 """
 
 
-@attr.s
 class CZMLWidget:
-    document = attr.ib(default=Document([Preamble()]))
```

Comment:
Why did you remove all these properties?

Reply:
The properties are in place. I have just made the Cesium Widget object creation task flexible.
Each property Change Elaborated below: 

1. Suppose I want to use the Cesium distribution files (CSS and JS scripts) of version 1.72.1, I could specify those while declaring the CZML Widget object If I do not specify, then the default is 1.79.1 (the current latest). 
In the previous approach, we were enforcing a specific version every time (1.68).
2. ion_token -  allowing users to specify an ion_token in case if they want to. Otherwise, creating an exception. ion_token is essential to overcome the 'invisible earth' problem. ion_token is used by the cesium API to fetch cesium assets needed for the plot. (in this case, the earth image). Cesium Ion API is free as of now.  
3. The terrain and imagery parameters give us the flexibility to specify the terrains and imageries of our choice (could be one of the various default terrains or could be one of our custom terrains). Earlier, with any plot, they were set to TERRAIN["Cesium"] and IMAGERY["Bing_Aerial"] by default. 
Based on my observations on the Cesium Ion website, we can upload our custom Tiles, Imageries, and 3D files on the Cesium cloud and then generate our custom plots. A very exciting feature that will unlock tremendous possibilities... I am yet to explore more on this.
4. _container_id parameter: This is a completely optional parameter. If specified, the CesiumWidget plot will be displayed in a div with id="CesiumContainer-custom_specified_name" otherwise, "CesiumContainer-name_specified_by_uuid4". No specific purpose or advantage in doing this.

To summarize, the properties are all the same. But, instead of using defaults, I have given the user extra flexibility in case if he/she wants to customize the plot.
With the new approach, only the ion_token becomes a compulsory field, which eliminates the 'invisible earth' issue.

Reply:
Even if "the properties are all the same", the fact that they are not declared in the body will confuse attrs. Please have a look at attrs documentation, especially https://www.attrs.org/en/stable/init.html?highlight=__attrs_post_init__#post-init-hook if you need it.

Also, the quality checks have failed, please do `tox -e reformat`.

Reply:
Sure. I will have a look at it. It seems I am misunderstanding attrs. I will work on the quality checks too. Thanks a lot.